lib/public/views/Runs/Overview/exportRunsButton.js

```diff
@@ -109,3 +109,14 @@ export const exportRunsModal = (runsOverviewModel, modalHandler) => {
         }),
     ]);
 };
+
+/**
+ * Builds a button which will redirect the user to the export run page
+ * @param {RunsOverviewModel} runsModel global model to access functions
+ * @param {ModelModel} modalModel global model to access functions
+ * @returns {Component} with button
+ */
+export const exportRunsButton = (runsModel, modalModel) => h('button.btn.btn-primary.w-15.h2.mlauto#export-runs-trigger', {
+    disabled: (runsModel.runs?.payload?.length ?? 0) === 0,
```

Comment:
this syntax is to ban (trying to get payload of remote data), it should be (considering that this PR is merged https://github.com/AliceO2Group/WebUi/pull/2219):
```js
disabled: !runsModel.runs?.match({
    Success: (payload) => payload.length,
    Other: () => null,
});
```

Reply:
refactored, but `Other` will be used when mentioned PR is merged